dsc/docs-conceptual/dsc-3.0/concepts/configurations.md

```diff
@@ -6,7 +6,201 @@ description: >-
+  validate this data or use it directly. The annotations can include notes like who authored the
+  document, the last time someone updated it, or any other information. DSC doesn't use the
+  annotations. The metadata is for documentation or other tools to use.
+- `parameters` (optional) - Defines a set of runtime options for the configuration. Parameters can
+  be referenced by resource instances to reduce duplicate definitions or enable dynamic values.
+  Parameters can have default values and can be set on any configuration operation.
+- `variables` (optional) - Defines a set of reusable values for the configuration. Variables can be
+  referenced by resource instances to reduce duplicate definitions.
+
 > [!NOTE]
-> The team is rewriting this document for DSCv3.
+> DSC isn't implemented to process the `parameters` and `variables` properties yet. They can be
+> defined in a document, but not referenced.
+
+## Defining a configuration document
+
+Minimally, a configuration document defines the `$schema` and `resources` properties. The
+`$schema` property must be a valid URI for DSC's configuration document schema. The `resources`
+property must define at least one DSC Resource instance.
+
+For example:
+
+```yaml
+# example.dsc.config.yaml
+$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/08/config/document.json
+resources:
+  - name: example key value
+    type: Microsoft.Windows/Registry
+    properties:
+      keyPath:   HKCU\example\key
+      valueName: Example
+      valueData:
+        String: This is an example.
+```
+
+The example document declares a single resource instance called `example key value`. The instance
+uses the `Microsoft.Windows/Registry` resource to declare that:
+
+- The `example\key` registry key should exist in the system's current user hive.
+- The `example\key` registry key should have a value called `Example`.
+- The `Example` value should be the string `This is an example`.
+
+The example document is _declarative_. It describes the desired state, not how to put the system
+into that state. It relies on the `Microsoft.Windows/Registry` resource to handle getting, testing,
+and setting the state of the registry key and value.
+
+### Defining resource instances
+
+As shown in the prior example, configuration documents include a collection of resource instances.
+Together, the instances describe a system's desired state. A configuration document can include any
+number of resource instances. The instances can be of any available DSC Resource.
+
+A resource instance declaration always includes:
+
+- `name` - A short, human-readable name for the instance that's unique in the document. This name
+  is used for logging and it helps to document an instance's purpose in the document.
+- `type` - The [fully qualified type name][02] for the resource to identify the resource DSC should
+  use to manage the instance.
```

Comment:
```suggestion
  is used for logging and it helps describe the purpose of the instance.
- `type` - The [fully qualified type name][02] of the resource that DSC should use to manage the instance.
```

Are these 2 uses of "instance" talking about the same thing?

Reply:
Yes - the _instance_ is the arbitrary component being managed. It's defined in three parts:

- The `properties` keyword defines the desired state of the instance.
- The `type` keyword defines the resource that processes the instance. The processing includes both property validation and operation invocation.
- The `name` keyword defines an annotation for human readers and users of the configuration. The name is included in logging for the instance and serves as minimal documentation for the instance.